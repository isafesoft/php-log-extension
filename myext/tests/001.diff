001+ PHP Warning:  Module 'myext' already loaded in Unknown on line 0
002+ 
003+ Warning: Module 'myext' already loaded in Unknown on line 0