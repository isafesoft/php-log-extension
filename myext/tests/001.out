PHP Warning:  Module 'myext' already loaded in Unknown on line 0

Warning: Module 'myext' already loaded in Unknown on line 0
myext extension is available